src/main/webapp/forms/noarray.gnd.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page import="de.uni_tuebingen.ub.nppm.db.*" isThreadSafe="false" %>
+
+<%
+    if (feldtyp.equals("gnd") && !array) {
+
+        String gndId = AbstractBase.getStringNative("SELECT " + zielAttribut + " FROM " + zielTabelle + " WHERE ID=\"" + id + "\"");
+
+        if (gndId != null && !gndId.trim().equals("")) {
+
+        String link = "<a href=\"https://d-nb.info/gnd/" + DBtoHTML(gndId) + "\" style=\"display: inline-block; line-height: 40px; vertical-align: middle; margin-top: 10px;\"> " + gndIcon + "</a>";
```

Comment:
Since this is a link to an external page, we should also add `target="_blank"`.

Reply:
done